### DISCUSSION TITLE
How does Redis new licensing policy works with Medusa since it's no longer Open Source

### DISCUSSION THREAD

**Discussion by waldothedeveloper** [medusajs/medusa#6968] [Q&A] [answered]
I'm not knowleagable when it comes to software licenses, so I'm not sure how this change in Redis's software licensing will affect or not it's ability to be used in Medusa for commercial projects like e-commerce, things like that. 

Does anyone can shed some light on this?

**Comment by chareen3** [comment, marked as answer]
@waldothedeveloper The change in Redis's licensing may impact its use in commercial projects like Medusa for e-commerce. It's essential to review the terms of the Redis Source Available License (RSAL) and ensure compliance with its requirements. Alternatively, consider exploring alternative database solutions with more permissive licensing terms if needed.
